gadgets/trace_tcpconnect/program.bpf.c

```diff
@@ -9,6 +9,8 @@
 #include <bpf/bpf_tracing.h>
 #include <bpf/bpf_endian.h>
 
+#define MAX_EVENT_SIZE 512
```

Comment:
What about using the actual size of the struct as suggested in https://github.com/inspektor-gadget/inspektor-gadget/pull/2323#discussion_r1435262889?

Reply:
If we aren't setting this to the real value, I think it's better not to define it at all.

Reply:
I still think it's a bad idea to define this to a value that is not the exact one, that's very misleading. However, I won't block the PR for this.

Reply:
What about a comment `// Note: this is not the exact size of the event due to ...`?